src/platform/device.gni

```diff
@@ -111,6 +111,14 @@ declare_args() {
   }
 }
 
+declare_args() {
+  if (chip_persist_subscriptions) {
```

Comment:
does `chip_subscription_timeout_resumption = chip_persist_subscriptions` work ? 

some comments about why these are separate configurations may be useful.

Reply:
Done.